source/guides/non-denizen/resource-packs.md

```diff
@@ -0,0 +1,248 @@
+Resource Packs - Custom Items And Sounds
+-------------------
+
+This page will answer some common questions from programmers interested in creating visually custom items and adding new sounds into a resource pack. There are plenty of tutorials on creating your own resource pack, and less commonly how to implement custom model data and manage sounds within one. This guide primarily, details how to correctly format your resource pack, and then implement it into Minecraft using Denizen scripts.
+
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+### Inside The Root Directory
+Directory: `.minecraft\resourcepacks\MyResourcePack\`
+
+The main directory within your resource pack folder should contain both:
+
+`Assets` Folder - This is where all your files are placed.
+`pack.mcmeta` - This is how Minecraft knows what format your Resource Pack is.
+Optimally, you can also include:
+
+* pack.png - This is a `64x64` custom image for your pack!
+
+#### Example File: `pack.meta`
+Inside the `pack.meta` file, you will find this is formatted in a `json` format. You need two named strings: `pack_format` and `description`.
+Here is what it looks like inside:
+```json
+{
+   "pack": {
+      "pack_format": 5,
+      "description": "My Fancy Resource Pack"
+   }
+}
+```
+
+##### pack.meta File Key: `pack_format`
+This is the indicator to Minecraft what version this pack is.
+1 indicates versions `1.6` - `1.8`.
+2 indicates versions `1.9` - `1.10`,
+3 indicates versions `1.11` - `1.12`,
+4 indicates versions `1.13` - `1.14`,
+5 indicates versions `1.15` - `1.16`.
```

Comment:
Probably put \` around the 1/2/3/4/5.
Also make it a markdown list (prefix with `-`)